runtime/dancebox/src/lib.rs

```diff
@@ -719,10 +718,14 @@ impl RemoveParaIdsWithNoCredits for RemoveParaIdsWithNoCreditsImpl {
         let credits_for_2_sessions = 2 * blocks_per_session;
         para_ids.retain(|para_id| {
             // Check if the container chain has enough credits for producing blocks for 2 sessions
-            let credits = pallet_services_payment::BlockProductionCredits::<Runtime>::get(para_id)
+            let free_credits = pallet_services_payment::BlockProductionCredits::<Runtime>::get(para_id)
                 .unwrap_or_default();
 
-            credits >= credits_for_2_sessions
+            let (block_production_costs, _) = <Runtime as pallet_services_payment::Config>::ProvideBlockProductionCost::block_cost(para_id);
+            let regular_credits = Balances::free_balance(pallet_services_payment::Pallet::<Runtime>::parachain_tank(para_id.clone()))
+                .saturating_div(block_production_costs);
+
+            (free_credits as u128).saturating_add(regular_credits) >= credits_for_2_sessions.into()
```

Comment:
This is to support both the old and the new kind of credits. Do we want to support both? Why not migrate old credits to the parachain tank address?